x-pack/plugins/cases/public/client/attachment_framework/types.ts

```diff
@@ -12,10 +12,11 @@ import {
   CommentRequestPersistableStateType,
 } from '../../../common/api';
 import { Case } from '../../containers/types';
+import { SupportedUserActionTypes } from '../../components/user_actions/types';
 
 export interface AttachmentViewObject<Props = {}> {
-  type?: EuiCommentProps['type'];
-  timelineIcon?: EuiCommentProps['timelineIcon'];
+  type?: SupportedUserActionTypes;
```

Comment:
Reminder to myself that we need to remove this prop.

Reply:
I took a peek at the alerting changes as well, and LGTM.